### PR TITLE
adds various scarfs to the ooc store

### LINKED ISSUE #7365: Green striped scarf missing from beecoin shop
## Round ID:

39518

## Testmerges:

/

## Reproduction:

Simply open game preferences/beeshop/accessories

The Green striped scarf is missing from beecoin shop! blue and red are there. very sad for my olive haired character Dave.

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
title

## Why It's Good For The Game
more variety, strange inconsistency, closes #7365 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: new scarfs now in the ooc store
/:cl:

### CODE REVIEW

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (Penwin0) [inline _maps/_basemap.dm]: You commited this

**Comment by ivanmixo** [comment]
Should add all the scarf flavors while you're at it.

**Comment by itsmeow** [comment]
Update CL and PR title to match new changes please

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Update CL

**Comment by itsmeow** [comment]
GTM

**Comment by etherware-novice** [comment]
gtm?

**Comment by itsmeow** [comment]
good to merge

**Pull request merged**